docs/services.md

```diff
@@ -1,114 +1,185 @@
 
-```
-<servicename>.<service>.homeoffice.gov.uk
 
-web-portal.example-service.homeoffice.gov.uk
-```
 
 ## Application Composition
-The following containers are typically used in applications hosted on ACP:
 
-#### Louketo Proxy (formerly Keycloak Proxy/Gatekeeper)
-[Louketo Proxy](https://github.com/louketo/louketo-proxy): a proxy that can be used to protect applications with Keycloak.
-#### Nginx Proxy
-[Nginx Proxy](https://github.com/UKHomeOffice/docker-nginx-proxy): for TLS and proxying your application container.
-#### Cert Manager
-[Cert-Manager](how-to-docs/cert-manager.md): for obtaining internal and external certificates.
+Applications hosted on ACP typically use the following containers:
+
+  * [Keycloak Gatekeeper][gatekeeper]: protects applications using [Keycloak][keycloak]. Note that in the future ACP is going to deprecate its usage.
+  * [Nginx Proxy][nginx]: provides TLS and proxying for your application container
+  * [Cert-Manager][cert_mgr]: obtains internal and external certificates
+
+
 
 ## Logging
-Logging stack consists of [Elasticsearch](https://github.com/UKHomeOffice/docker-elasticsearch), [Fluentd](https://github.com/fluent/fluentd), [Kibana](https://github.com/UKHomeOffice/docker-kibana).
 
-- Fluentd agents deployed as a daemonSet will collect all workload logs and index them in Elasticsearch.
-- Logs are searchable for a period of 5 days through [Kibana UI](https://kibana.acp.homeoffice.gov.uk). Access to view logs can be requested via [Support request](https://support.acp.homeoffice.gov.uk/servicedesk/customer/portal/1/create/34).
+ACP’s logging stack consists of [Elasticsearch][elasticsearch], [Fluentd][fluentd], and [Kibana][kibana], also known as EFK:
+
+  * ACP deploys Fluentd agents as a daemonSet, which collect all workload logs and indexes them in Elasticsearch.
+  * you can search for logs on [ACP's Kibana instance][acp_kibana]. To access Kibana for your namespace, raise [this support request][support_kibana] on ACP Support.
+
+### Log Retention Policy
+
+The following applies both to workload logs and all logs within EFK:
+
+  * logs remain searchable in Kibana for 5 days and within Elasticsearch for 10 days.
+  * ACP persists workload logs it collects in S3 indefinitely  
+  * after 60 days, ACP migrates logs in S3 to the infrequent access storage class and then glacier storage  
+  * after 180 days, ACP migrates logs in S3 to glacier storage
+
+
+
+## Monitoring and Metrics
+
+### Sysdig
+
+ACP uses Sysdig for container monitoring and metric collection. Sysdig provides these through alert monitoring and metric dashboards for your Kubernetes infrastructure. You can access our instance [here][acp_sysdig].
+
+We are working with Sysdig to develop integration between our services. For more information, including the Sysdig Inspect command line tool, see the [Sysdig documentation][sysdig_docs] and [Sysdig Inspect][sysdig_inspect].
```

Comment:
I wasn't aware of this myself

Reply:
Will seek verification on this from @rickwilliams391

Reply:
I would take out the second paragraph - we are not actively working with Sysdig on service integration